homework/Kiet_Tran/lesson_4/insert_delete_getrandom_o1.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * https://leetcode.com/problems/insert-delete-getrandom-o1/
+ */
+class RandomizedSet {
+
+    private $set;
+
+    /**
+     */
+    function __construct() {
+        $this->set = [];
+    }
+  
+    /**
+     * @param Integer $val
+     * @return Boolean
+     */
+    function insert(int $val): bool {
+        if (in_array($val, $this->set)) {
+            return false;
+        }
+
+        array_push($this->set, $val);
+        return true;
+    }
+  
+    /**
+     * @param Integer $val
+     * @return Boolean
+     */
+    function remove(int $val): bool {
+        if (!in_array($val, $this->set)) {
+            return false;
+        }
+
+        $key = array_search($val, $this->set);
```

Comment:
what is the runtime of this operation?

Reply:
It is O(n)
I updated it with using hashmap
Thank you